src/DotNetTwitterBot/Function.cs

```diff
@@ -20,10 +21,12 @@ public async Task Retweet(ILambdaContext context)
 
             Auth.SetUserCredentials(creds.ConsumerKey, creds.ConsumerSecret, creds.AccessToken, creds.AccessSecret);
 
-            var searchTerms = new[] { "\".NET Framework\"", "\".NET Core\"", "\".NET 5\"", "dotnet", "dotnetcore", "_dotnetbot_" };
+            var searchTerms = new[] { $"\".NET Framework\"", "\".NET Core\"", "\".NET 5\"", "dotnet", "dotnetcore", "_dotnetbot_" };
```

Comment:
The `$` is not necessary here, there's no string interpolation being done.

Reply:
done